spec.html

```diff
@@ -39111,7 +39069,7 @@ <h1>GeneratorFunction.prototype.constructor</h1>
 
       <emu-clause id="sec-generatorfunction.prototype.prototype">
         <h1>GeneratorFunction.prototype.prototype</h1>
-        <p>The value of `GeneratorFunction.prototype.prototype` is the %Generator.prototype% intrinsic object.</p>
+        <p>The initial value of `GeneratorFunction.prototype.prototype` is the Generator Prototype Object.</p>
```

Comment:
Why this (and other similar) changes to no longer use the % % syntax? Most sections on Foo.prototype describe the initial value as %Foo.prototype%.

Reply:
I guess the reasoning is the the comment thread from above with @jmdyck.

I'm not a fan of changing the wording for these for just %Generator% and %AsyncGenerator% without changing all the other Foo.prototype sections.

Reply:
Sure, I'll update all the other sections as well.